command/build.go

```diff
@@ -284,6 +288,9 @@ Options:
   -except=foo,bar,baz        Build all builds other than these
   -force                     Force a build to continue if artifacts exist, deletes existing artifacts
   -machine-readable          Machine-readable output
+  -on-error=abort            If the build fails, abort without cleanup
+  -on-error=ask              If the build fails, prompt for action
+  -on-error=cleanup          If the build fails, clean up and exit (the default)
```

Comment:
I would prefer if they where on the same line, maybe:

`-on-error=[cleanup|ask|abort]       If the build fails do: clean up (default), ask, or abort`

I think it's clear enough and the details are in the docs.